configcatclient/localdictionarydatasource.py

```diff
@@ -0,0 +1,14 @@
+from .overridedatasource import OverrideDataSource
+from .constants import *
+
+
+class LocalDictionaryDataSource(OverrideDataSource):
```

Comment:
Shouldn't we provide a local file data source too for python? WDYT?

Reply:
I'll add it to this PR